src/renderer/components/+catalog/catalog.scss

```diff
@@ -0,0 +1,23 @@
+.CatalogPage {
+  --width: 100%;
+  --height: 100%;
+  text-align: left;
+
+  .content-wrapper {
+    .content {
+      margin: unset;
+      max-width: unset;
+      height: 100%;
+    }
+  }
+}
+
+.TableCell.status {
```

Comment:
Better to move inside some context (e.g. `.CatalogPage > .TableCell`), otherwise it's a global style for whole app.

Reply:
Fixed.